webui/layout.ego

```diff
@@ -17,9 +17,10 @@ func ego_layout(w io.Writer, req *http.Request, yield func()) {
 
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
 
-    <link href="<%= relative(req, "/static/bootstrap.css") %>" media="screen" rel="stylesheet" type="text/css" />
```

Comment:
Bootstrap is always required. rtl overrides some of the base classes.

Reply:
Bootstrap 5 has it's own RTL, not a 3rd party CSS. The RTL CSS is a complete replacement, not an addition. See: https://getbootstrap.com/docs/5.0/getting-started/rtl/